### ISSUE TITLE
ls return code is not zero

### ISSUE THREAD

**Issue by ponty** [yangh/sevensquare#27]
My phone is not rooted.

```
$ ls -d /sys/class/input/input*                                                                                                                                                                         
ls: /sys/class/input/input0: Permission denied
ls: /sys/class/input/input1: Permission denied
ls: /sys/class/input/input2: Permission denied
ls: /sys/class/input/input3: Permission denied
ls: /sys/class/input/input4: Permission denied
```
`ls` return code is not zero, so I had to remove the line `emit deviceDisconnected()` in code:
```c
    args << "shell" << "ls -d" << QString(SYS_INPUT_DIR) + "input*";
    adb.run(args);

    if (! adb.exitSuccess()) {
        emit deviceDisconnected();
        return;
    }
```

Now the program works without disconnection.

**Comment by stridjj** [comment]
where is this file in which you remove "emit deviceDisconnected();" ?

**Comment by stridjj** [comment]
found myself:
/path/to/sevensquare/src/adbfb.cpp

**Comment by yangh** [comment]
Fixed in latest commit 1333e50

For Andriod user build, there is no permission to do so.
